networkx/algorithms/centrality/harmonic.py

```diff
@@ -31,10 +30,12 @@ def harmonic_centrality(G, distance=None):
     Parameters
     ----------
     G : graph
-      A NetworkX graph.
+      A NetworkX graph
+    u : nodes, optional
+      Return only the value for nodes u
     distance : edge attribute key, optional (default=None)
-      Use the specified edge attribute as the edge distance in shortest path
-      calculations. If `None`, then each edge will have distance equal to 1.
+      Use the specified edge attribute as the edge distance in shortest
+      path calculations
```

Comment:
What happens when `distance=None`? The docs should tell the user what to expect.